settings.py

```diff
@@ -0,0 +1,103 @@
+import os
+from read_env import read_env
+
+# Read .env into os.environ (if it exists)
+try:
+    read_env()
+except FileNotFoundError:
+    pass
+
+domains = {
+    'stage1': {
+        'home': 'https://staging.osf.io',
+        'api': 'https://staging-api.osf.io',
+        'files': 'https://files.us.staging.osf.io',
+        'cas': 'https://staging-accounts.osf.io'
+    },
+    'stage2': {
+        'home': 'https://staging2.osf.io',
+        'api': 'https://staging2-api.osf.io',
+        'files': 'https://files.us.staging2.osf.io',
+        'cas': 'https://staging2-accounts.osf.io'
+    },
+    'stage3': {
+        'home': 'https://staging3.osf.io',
+        'api': 'https://staging3-api.osf.io',
+        'files': 'https://files.us.staging3.osf.io',
+        'cas': 'https://staging3-accounts.osf.io'
+    },
+    'test': {
+        'home': 'https://test.osf.io',
+        'api': 'https://test-api.osf.io',
+        'files': 'https://files.us.test.osf.io',
+        'cas': 'https://test-accounts.osf.io'
+    },
+    'prod': {
+        'home': 'https://osf.io',
+        'api': 'https://osf-api.io',
+        'files': 'https://files.us.osf.io',
+        'cas': 'https://accounts.osf.io'
+    }
+}
+
+DRIVER = os.environ.get('DRIVER', 'Firefox')
+HEADLESS = os.environ.get('HEADLESS', False)
+
+QUICK_TIMEOUT = os.environ.get('QUICK_TIMEOUT', 4)
```

Comment:
Drive-by: environment variables will always be strings, `os.environ` won't do any sort of type-casting for you. You would have to do something like

```python
QUICK_TIMEOUT = int(os.environ.get('QUICK_TIMEOUT', 4))
```

You could use a [environs](https://github.com/sloria/environs), which will do type-casting for you:

```python
from environs import Env
env = Env()
env.read_env()

QUICK_TIMEOUT = env.int('QUICK_TIMEOUT', default=4)
```

It'll also allow you to have required envvars. Just omit the `default` argument.

```python
# no default given, so this will error if USER_ONE_PASSWORD
# is unset (that's a good thing)
USER_ONE_PASSWORD = env.str('USER_ONE_PASSWORD')
```

Full disclosure: I'm the author of environs.

Reply:
Ahhh good catch. Thanks, Steve!